src/Core/Utils.fs

```diff
@@ -0,0 +1,6 @@
+module Utils
+
+let boolOptionToBool opt = 
```

Comment:
Looks like we are actually not using this function - should we remove it, together with the file?